cachito/web/models.py

```diff
@@ -340,11 +340,7 @@ def content_manifest(self):
         return content_manifest.ContentManifest(self, packages)
 
     def _is_complete(self):
-        if len(self.states) > 0:
-            latest_state = self.states[-1]
-            return latest_state.state_name == RequestStateMapping.complete.name
-
-        return False
+        return self.state is not None and self.state.state_name == RequestStateMapping.complete.name
```

Comment:
@brunoapimentel IIRC, wasn't this one of the causes of the empty packages bug?

Reply:
Yeah. Sorry. I'll revert this change.

Reply:
Yeah, we need to keep using the `self.states` attribute. This might be  a good reason to try to prioritize the story that covers the refactoring of this dual relationship.